packages/loader/container-loader/src/container.ts

```diff
@@ -650,10 +654,11 @@ export class Container extends EventEmitterWithErrorHandling<IContainerEvents> i
 
         this._deltaManager = this.createDeltaManager();
         this.storage = new ContainerStorageAdapter(
-            () => {
+            (callName: string) => {
                 if (this.attachState !== AttachState.Attached) {
                     this.logger.sendErrorEvent({
                         eventName: "NoRealStorageInDetachedContainer",
+                        callName,
```

Comment:
I feel stack is sufficient to identify the offender (public method names are not renamed by bundler / minimizer), and in most cases name alone (without stack) is not enough to figure out the cause.